seedbuilder/areas.ori

```diff
@@ -1793,16 +1793,24 @@ home: MoonGrottoAboveTeleporter
 	pickup: LeftGrottoTeleporterExp
 		casual-core WallJump DoubleJump
 		casual-core Climb DoubleJump ChargeJump
+		standard-abilities DoubleJump Dash Ability=3
 		expert-core Grenade Bash
 		expert-core Climb ChargeJump
+		expert-core DoubleJump Dash
 		expert-dboost ChargeJump Health=4
 		expert-dboost WallJump Glide Health=4
 		expert-dboost Climb Glide Health=4
 		expert-dboost DoubleJump Glide Health=4
 		expert-dboost WallJump Health=7
 		expert-dboost Climb Health=7
+		expert-dboost Climb DoubleJump Health=4
+		expert-dboost Climb Dash Ability=3 Health=4
+		expert-dboost WallJump Dash Ability=3 Health=4
+		expert-dboost Dash Ability=6 Health=4
 		master-lure Bash
 		master-dboost DoubleJump Health=4
```

Comment:
This path can be lowered to `expert-dboost`, based on our double jump wall climbing rules. This is as straightforward as pressing jump again after faceplanting the spikes.